packages/destination-actions/src/destinations/pipedrive/pipedriveApi/organizations.ts

```diff
@@ -0,0 +1,27 @@
+import { RequestClient } from "@segment/actions-core/src/create-request-client"
+
+export interface Organization {
+  name?: string
+  add_time?: string
+}
+
+export async function createOrUpdateOrganizationById(
+  request: RequestClient,
+  domain: string,
+  organizationId: number | null,
+  organiztion: Organization,
```

Comment:
Typo:
```suggestion
  organization: Organization,
```

(if applied, you'll need to fix references in the function body too)